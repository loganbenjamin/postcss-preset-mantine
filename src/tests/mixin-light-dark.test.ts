const { testTransform } = require('./utils');

const baseInput = `
.demo {
  @mixin light {
    color: red;
  }

  @mixin dark {
    color: blue;
  }
}
`;

describe('mixin-light-dark', () => {
  it('should work', async () => {
    const res = await testTransform(baseInput);
    expect(res.css).toMatchSnapshot();
  });
});
