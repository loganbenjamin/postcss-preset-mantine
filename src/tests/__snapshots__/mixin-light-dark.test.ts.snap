// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mixin-light-dark should work 1`] = `
"
[data-mantine-color-scheme='light'] .demo {
    color: red
}
  [data-mantine-color-scheme='dark'] .demo {
    color: blue
}
"
`;
